models/stock_quant.py

```diff
@@ -63,14 +63,17 @@ def total_quantity(self):
         """
         return sum(self.mapped('quantity'))
 
-    def group_quantity_by_product(self):
+    def group_quantity_by_product(self, only_available=False):
         """ Returns a dictionary with the total quantity per product,
             mapped by product_id.
         """
         products = {}
         for quant in self:
             products.setdefault(quant.product_id.id, 0)
```

Comment:
It setting products as a defaultdict to begin with removes this line.

Reply:
OK